.vscode/launch.json

```diff
@@ -4,17 +4,24 @@
     // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
```

Comment:
Do we really want .vscode in the project? It feels like something that should be removed in .gitignore..

Reply:
We looked at this a little while ago and thought it would be useful for devs running the example project so they wouldn't have to set anything up, whilst also not polluting the repo too much. 

Do you think IDE-specific files should always be excluded?

Reply:
Yes, it's better to have instructions in the README on how to set it up in different IDEs (if there is any difficulty). The problem with having ide specific files is that not everybody runs the same IDE, and it's possible that a different IDE might clash with these. Also these files will be useless clutter to a sizable percentage of users of the SDK.

Reply:
Would you feel it would be ok if `.vscode` lived in the /examples folder instead? That seems like less clutter, and would still operate in the same way

Reply:
I'm a bit confused why it's so important to get it in? Are there any particular difficulty getting vs code to work?

Reply:
It just means that if you you try to debug from anywhere but the file you're wanting to run, it will show this error and create the `launch.json` file and `.vscode` folder for you if you click `open launch.json`
![image](https://user-images.githubusercontent.com/1381991/37591189-9801fcd8-2b62-11e8-8ba2-6174e0380a1f.png)
Where as if you have the folder and file, it gives you dropdowns for the different run configurations from anywhere.

Reply:
@leonard-kramer ok Seb made a good suggestion - we can add a `CONTRIBUTING.md` which can have this launch.json file in a gist. And I can `git rm` the folder, and add it to `.gitignore`, sound ok?

Reply:
yeah, that sounds like a better way.